<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inicio de Sesión</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* === Ventana Principal === */
QWidget {
	background-color: #3c3c3c; /* Fondo gris oscuro */
}

/* === Tarjeta de Login === */
QGroupBox {
	background-color: #4a4a4a; /* Gris medio para la &quot;tarjeta&quot; */
	border: 1px solid #5a5a5a;
	border-radius: 8px;
	margin-top: 10px;
}

QGroupBox::title {
	subcontrol-origin: margin;
	subcontrol-position: top center;
	padding: 0 3px;
	color: #dddddd; /* Color de texto claro */
	font-weight: bold;
}

/* === Etiquetas (&quot;Usuario:&quot;, &quot;Contraseña:&quot;) === */
QLabel {
	font-size: 10pt;
	color: #dddddd;
	font-weight: bold;
	background-color: transparent;
}

/* === Campos de texto === */
QLineEdit {
	font-size: 10pt;
	padding: 5px;
	background-color: #505050;
	color: #e0e0e0;
	border: 1px solid #666666;
	border-radius: 4px;
}

QLineEdit:focus {
	border: 1px solid #888888;
}

/* === Botón de &quot;Ingresar&quot; (Azul) === */
QPushButton {
	font-size: 10pt;
	font-weight: bold;
	padding: 8px;
	background-color: #0078d7; /* Mantenemos el azul como acento */
	color: white;
	border: none;
	border-radius: 3px;
}

QPushButton:hover {
	background-color: #005a9e; /* Azul más oscuro */
}

/* === Label de Estado (Error) === */
#label_status {
	color: #e53935; /* Un rojo más brillante para modo oscuro */
	font-size: 9pt;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Iniciar Sesión</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Usuario:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="line_edit_usuario"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Contraseña:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="line_edit_password">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="boton_ingresar">
        <property name="text">
         <string>Ingresar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_status">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
